### ISSUE TITLE
Having multiple axis inputs bound on the same linked controller causes them to incorrectly override each other

### ISSUE THREAD

**Issue by Cottor-OwO** [getItemFromBlock/Create-Tweaked-Controllers#35]
While playing on 1.20.1 1.2.4 I discovered that linking multiple controller buttons to a single link causes the latter to only react to one of the buttons bound. For example:

1.  Here I bound the link to -Y first, then to +Y and lastly to RT:
![Screenshot 2024-09-28 232700](https://github.com/user-attachments/assets/63d78372-ab64-45ce-870b-a7acc0911be4)

2. This is no buttons pressed:
![2024-09-28_23 27 14](https://github.com/user-attachments/assets/bbfa4d8e-cd60-483d-aa99-f72d8028c5bb)

3. This is RT pressed:
![2024-09-28_23 27 24](https://github.com/user-attachments/assets/a77c444e-26e6-4c8a-9bbf-10abafdb270c)

4. This is -Y pressed: 
![2024-09-28_23 27 31](https://github.com/user-attachments/assets/6f76c5c3-41b9-43d5-bf80-f282fede2495)

5. This is +Y pressed:
![2024-09-28_23 27 34](https://github.com/user-attachments/assets/36a75e89-88df-4bf4-8700-8fea2b4e0d31)

This is unlike the Create controller which is capable of such a thing.

I use keyboard if this info is relevant.

Edit: I've just read another issue where this is mentioned, so this issue may be redundant.

**Comment by getItemFromBlock** [comment]
Yes, this is related to a part of what https://github.com/getItemFromBlock/Create-Tweaked-Controllers/issues/34 describes
The problem is that axis inputs don't behave the same way than button inputs do, for various technical reasons. I will try to make it behave more like buttons in a future update, so that cases like what you are doing will behave correctly. For now the best thing to do is to only bind button inputs to the redstone link when you want to bind multiple inputs on it

**Comment by getItemFromBlock** [comment]
I will still mark this one as a bug, because the other issue talking about it reports two things at once and this problem is not the first thing reported

**Comment by getItemFromBlock** [comment]
Fixed by commit ef1a76e24a579a07c4d411ad9b9dc5a8d241e9c1

The new behavior takes the maximum value instead of the one from the last button or axis

**Comment by getItemFromBlock** [comment]
Fixed with latest release
